File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        int answer = 0;
        for (String i : order) {
            if (i.contains("cafelatte")) {
                answer += 5000;
            } else {
                answer += 4500;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.kt
class Solution {
    fun solution(order: Array<String>): Int {
        var answer: Int = 0
        for (i in order){
            if (i.contains("cafelatte")){
                answer += 5000
            } else {
                answer += 4500
            }
        }
        return answer
    }
}